src/horus_api.h

```diff
@@ -35,20 +35,78 @@
 #include <stdint.h>
 #include "modem_stats.h"
       
-#define HORUS_MODE_BINARY            0
-#define HORUS_MODE_RTTY              1
+/* Horus API Modes */
+#define HORUS_MODE_BINARY_V1            0  // Legacy binary mode
+#define HORUS_MODE_BINARY_V2_256BIT     1  // New 256-bit LDPC-encoded mode
+#define HORUS_MODE_BINARY_V2_128BIT     2  // New 128-bit LDPC-encoded mode
+#define HORUS_MODE_RTTY                 99 // RTTY Decoding
+
+
+// Settings for Legacy Horus Binary Mode (Golay Encoding)
+#define HORUS_BINARY_V1_NUM_CODED_BITS             360
+#define HORUS_BINARY_V1_NUM_UNCODED_PAYLOAD_BYTES  22
+#define HORUS_BINARY_V1_DEFAULT_BAUD               100
+#define HORUS_BINARY_V1_DEFAULT_TONE_SPACING       270  // This is the minimum tone spacing possible on the RS41 
+                                                        // reference implementation of this modem.
+                                                        // Note that mask estimation is turned off by default for 
+                                                        // this mode, and hence this spacing is not used.
+
```

Comment:
I'm still uncomfortable with this. @darksidelemm the idea of an API is to abstract out hardware, not hard code it in. I feel this belongs in the calling application (demo program or freedv-gui). This feels more like a decision made for a specific application, than an API.  If that's the case (a one-off application with no general use case) - then lets talk about moving ```horus_api.*``` out of codec2 and into the Horus repo.

Reply:
The point of this was to provide some sensible default value which can be used if nothing else was provided by the user. I thought we may as well make it something that might provide some improvement in the current use-case of this modem, and it made sense to keep all these parameters together in one place, rather than splitting it up across multiple files.

By your argument, should we even be specifying a baud rate here? That's could be considered an implementation specific choice too...